modules/gapi/test/test_precomp.hpp

```diff
@@ -16,8 +16,9 @@
 #include <opencv2/ts.hpp>
 
 #include <opencv2/gapi.hpp>
-#include <opencv2/gapi/imgproc.hpp>
 #include <opencv2/gapi/core.hpp>
+#include <opencv2/gapi/imgproc.hpp>
+#include <opencv2/gapi/video.hpp>
```

Comment:
`#ifdef HAVE_OPENCV_VIDEO` is required here.

Try to build OpenCV with disabled video module:
```
cmake -DBUILD_opencv_video=OFF ...
```

Reply:
Yeah, I've just checked it now (just in case, because I've already build it before): it compiles without opencv_video as there is a workaround on that case. There is no dependency on opencv_video in `opencv2/gapi/video.hpp` because it is a declaration of kernels, and the CPU(OpenCV) implementation (in [modules/gapi/src/backends/cpu/gcpuvideo.cpp](https://github.com/opencv/opencv/pull/16788/files#diff-baf3285f744f48afaa6774d88c1c4ba1) is protected by `#ifdef`

Reply:
opencv2/gapi/video.hpp file doesn't contain information which depends on video module, as I got. It just contains G-API operations interfaces.

Reply:
Sorry, my mistake - read as `opencv2/video.hpp`